netlink/conntracker.go

```diff
@@ -45,10 +45,12 @@ type realConntracker struct {
 	statsTicker   *time.Ticker
 	compactTicker *time.Ticker
 	stats         struct {
-		gets               int64
-		getTimeTotal       int64
-		registers          int64
-		registersTotalTime int64
+		gets                 int64
+		getTimeTotal         int64
+		registers            int64
+		registersTotalTime   int64
+		unregisters          int64
```

Comment:
Where are these incremented?

Reply:
They should be incremented in `unregister` but there was a typo